### ISSUE TITLE
add a user guide on configuring a File.client

### ISSUE THREAD

**Issue by edublancas** [ploomber/ploomber#485]
note: add link from soopervisor docs to it

**Comment by idomic** [comment]
We can do this as part of th kflow submit sounds pretty straight forward

**Comment by edublancas** [comment]
yes. this is repeated in several places in the soopervisor config. I'm thinking to concentrate a simple explanation in soopervisor, and have another more detailed section in ploomber docs

**Comment by idomic** [comment]
Good idea! Like for me I took an example for the gcp client inputs so it
was pretty straight forward but maybe since ei had the context.

**Comment by edublancas** [comment]
[added guide to soopervisor docs,](https://soopervisor.readthedocs.io/en/latest/user-guide/task-comm.html) we need something like this but with more details in ploomber docs
